xformers/sparse/blocksparse_tensor.py

```diff
@@ -114,8 +114,7 @@ def __init__(self, values, layout):
     def __repr__(self):
         return f"block_sparse_tensor(shape={self.shape}, values={self.__values})"
 
-    @property
-    def _blocksparse_values(self):
+    def values(self):
         return self.__values
```

Comment:
We now unify obtaining the values of the sparse tensor under the same API, which follows the API from PyTorch